src/lib/mia_platform_client.py

```diff
@@ -37,122 +38,132 @@ def __init__(self):
         self.session = requests.Session()
         self.session.auth = MiaPlatformAuth()
 
+        logging.basicConfig(
```

Comment:
I see we are using the same logging configuration for the app logger, maybe we can move this to a constant somewhere?